__tests__/app.test.js

```diff
@@ -127,6 +127,140 @@ describe("GET Endpoints", () => {
+        .expect(200)
+        .then((response) => {
+          const { articles } = response.body;
+          expect(articles).toBeInstanceOf(Array);
+          expect(articles).toHaveLength(12); // 11 mitch articles
+          expect(articles).toBeSorted({ key: "title", descending: true });
+          articles.forEach((article) => {
+            expect(article).toMatchObject({
+              author: expect.any(String),
+              topic: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(String),
+            });
+          });
+        });
+    });
+  });
+
+  describe("GET /api/articles (10) filtered by queries", () => {
+    it("200: Returns an array of objects containing articles sorted by column > title", () => {
+      return request(app)
+        .get("/api/articles?sort_by=title&order=asc") // has one cat article
+        .expect(200)
+        .then((response) => {
+          const { articles } = response.body;
+          expect(articles).toBeInstanceOf(Array);
+          expect(articles).toHaveLength(12); // 11 mitch articles
+          expect(articles).toBeSorted({ key: "title", ascending: true });
+          articles.forEach((article) => {
+            expect(article).toMatchObject({
+              author: expect.any(String),
+              topic: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(String),
+            });
+          });
+        });
+    });
+  });
+
+  describe("GET /api/articles (10) filtered by queries", () => {
+    it("200: Returns an array of objects containing articles topic=mitch, sort_by=votes, order=asc", () => {
+      return request(app)
+        .get("/api/articles?topic=mitch&sort_by=votes&order=asc") // has one cat article
+        .expect(200)
+        .then((response) => {
+          const { articles } = response.body;
+          expect(articles).toBeInstanceOf(Array);
+          expect(articles).toHaveLength(11); // 11 mitch articles
+          expect(articles).toBeSorted({ key: "votes", ascending: true });
+          articles.forEach((article) => {
+            expect(article).toMatchObject({
+              author: expect.any(String),
+              topic: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(String),
+            });
+          });
+        });
+    });
+  });
+
+
+
```

Comment:
Great testing so far, can i just see a test to prove that if i pass order=desc it still works.
Also do you think your code can handle both desc and DESC, will this matter to your api, will your client  have to make this distinction?